packages/io/svg-deserializer/src/shapesMapGeometry.js

```diff
@@ -437,7 +436,7 @@ const expandPath = (obj, svgUnitsPmm, svgUnitsX, svgUnitsY, svgUnitsV, svgGroups
           }
         } else if (pathSelfClosed === 'split') {
           newPath()
-        } else {
+        } else if(coNext){ // allow self close in the last command #1135
```

Comment:
@hrgdavor I know that this is correct syntax, however the addition of if(coNext) expression hides the fact this this is the case where pathSelfClosed === 'error'. I had to review the code in order to understand this. So, I suggest the following...
- Add a comment that the else clause is for 'error'
- Add a nested if statement to test if(coNext)

Reply:
I have changed it bit more, check if this variant is more understandabla